README.md

```diff
@@ -95,7 +97,9 @@ The `CommonFactory` reads a RabbitMQ configuration from the rabbitMQ.json file.
   "minConnectionRecoveryTimeout": 10000,
```

Comment:
Could you please split configuration description to mq_router.json and rabbitMQ.json

Reply:
done